src/components/Cart/CartButtons.jsx

```diff
@@ -0,0 +1,29 @@
+import { useCart } from "../../context";
+import { removeFromCartHandler, updateCartHandler } from "../../utils";
```

Comment:
Not mandatory but good to know: You can use path alias to avoid moving back in folders using `../../`
refer For `Create React App without using Craco` section in this https://dev.to/theodorusclarence/clean-up-your-imports-using-absolute-imports-alias-on-react-apps-next-js-cra-and-react-vite-2160

Reply:
Thanks, for sharing. Will implement this